### PR TITLE
libraries/podofo: Update for 0.10.3 (+new maintainer)

### PR DESCRIPTION
**Pull request by isaackwy**
Larry Hajali is no longer maintaining SlackBuilds.

I would like to take over maintenance of podofo.
My overall goal is to take over maintenance of geopandas within the next few weeks.

### CODE REVIEW

File: libraries/podofo/podofo.SlackBuild [hunk 1]
```diff
@@ -80,36 +81,28 @@ find -L . \
  \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \+
 
-sed -i "s/BINARY_DIR}\/objects\"/BINARY_DIR}\/objects\" || true/" test/TokenizerTest/CMakeLists.txt
-
-# Fix mandir directory.
-sed -i 's|share/man|man|' CMakeLists.txt
-# Ref: https://sourceforge.net/p/podofo/tickets/28/
-patch -p1 < $CWD/XRefStrParObj-deprecated.patch
```
Comment (aclemons) [inline libraries/podofo/podofo.SlackBuild:88]: Delete the actual patch file too?

Comment (isaackwy) [reply libraries/podofo/podofo.SlackBuild:88]: Thanks Andrew - I now have deleted the patch file.

**Comment by willysr** [comment]
@sbo-bot: build libraries/podofo

**Comment by willysr** [comment]
merged

**Pull request closed without merging**